runtimes/common/src/constants.rs

```diff
@@ -152,7 +152,7 @@ pub mod attestation {
 	}
 }
 
-pub mod asset_switch {
+pub mod pallet_asset {
```

Comment:
Just `assets` I think is best.

Reply:
assetS

Reply:
[538c28a](https://github.com/KILTprotocol/kilt-node/pull/701/commits/538c28aacd45ec2a24cdb0737f7a44bbf100e27f)